contracts/scripts/deploy.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
-set -euox pipefail
+# set -x
```

Comment:
leftover?

Reply:
I understand -x as a debugging flag, as it is rather noisy. I can remove or leave it commented out